src/Components/SavedSearchAlert/ConfirmationArtworksGrid.tsx

```diff
@@ -0,0 +1,104 @@
+import { Flex, SkeletonText, Spacer, Text } from "@artsy/palette"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { FC } from "react"
+import { graphql } from "react-relay"
+import {
+  ConfirmationArtworksGridQuery,
+  ConfirmationArtworksGridQuery$data,
+} from "__generated__/ConfirmationArtworksGridQuery.graphql"
+import ArtworkGrid, {
+  ArtworkGridPlaceholder,
+} from "Components/ArtworkGrid/ArtworkGrid"
+import { SearchCriteriaAttributes } from "Components/SavedSearchAlert/types"
+import { useTranslation } from "react-i18next"
+
+interface ConfirmationArtworksProps {
+  artworksConnection: ConfirmationArtworksGridQuery$data["artworksConnection"]
+}
+
+const ConfirmationArtworks: FC<ConfirmationArtworksProps> = ({
+  artworksConnection,
+}) => {
+  const { t } = useTranslation()
+  const artworksCount = artworksConnection?.counts?.total
+
+  return (
+    <Flex flexDirection="column">
+      <Text variant="sm-display" color="black60">
+        {t("createAlertModal.confirmationStep.artworksMatchCriteria", {
+          artworksCount,
+        })}
+      </Text>
+      <Text variant="sm-display" color="black60">
+        {t("createAlertModal.confirmationStep.seeOurTopPicks")}
+      </Text>
+
+      <Spacer y={2} />
+
+      <ArtworkGrid artworks={artworksConnection!} columnCount={2} />
```

Comment:
I might need your help here. Grid doesn't render correctly in the confirmation modal for some reason (mWeb). I cannot figure out how to fix it properly

https://github.com/artsy/force/assets/3934579/272dc1f3-4e80-4358-9770-d532ac9ca2d9

Reply:
That component is a bit of a mess and maybe overkill. Can you just use the `Masonry` component?

Reply:
Masonry has the same problem, unfortunately..

Reply:
[Temporarily changed](https://github.com/artsy/force/pull/12543/commits/ec582e8748d78601d27ed65f5057d2ace4ae809d) to GridColumns which works fine, but completely differs from design specs. 

Still looking for a proper solution for the ArtworksGrid

Reply:
I can take a look at this today as well

Reply:
btw @gkartalis pointed to how Masonry is used together with GridColumns: https://github.com/artsy/force/blob/2f7217c3daa9bb8f2d8164408384d3ceec5cb787/src/Apps/Artist/Routes/Consign/Components/ArtistConsignFAQ.tsx#L26

I've re-used the same approach and it worked! Going to read about css grids this weekend...

and props to @gkartalis ❤️